graylog2-server/src/main/java/org/graylog/events/processor/EventProcessorConfig.java

```diff
@@ -74,12 +74,15 @@ default Set<String> requiredPermissions() {
     }
 
     /**
-     * Returns whether this config type is allowed to be exported in a Content Pack.
+     * Returns whether this config type should be presented to users,
+     * as opposed to being used for background functionality.
      *
-     * @return whether the config type can be exported in a Content Pack
+     * i.e. Should this be able to be exported in a Content Pack or modified on the Event Definitions Page
+     *
+     * @return whether the config type should be presented to users
      */
     @JsonIgnore
-    default boolean isContentPackExportable() {
+    default boolean isUserPresentable() {
```

Comment:
This is the check that was added to prevent users from exporting Sigma configs in Content Packs. I repurposed it to be more general since it seems like they really cover the same use case: whether or not this event processor type is meant to just serve a background purpose. But I'm open to suggestions if anyone sees issues with this.

Also open to suggestions on the naming/wording.

Reply:
This works well for this particular use case, but I wonder if there might ever be a case in the future where we want the ability to have `isContentPackExportable` and `isUserPresentable` be different. I can't think of a concrete example but we may want to add a separate method instead of combining the two.

Reply:
Ok I think that makes sense. It certainly wont hurt anything to separate out the logic so I'll go ahead and separate out into two functions.